src/components/templates/ConversationSpace/ConversationSpace.tsx

```diff
@@ -18,19 +21,38 @@ import TableHeader from "components/molecules/TableHeader";
 import { UserList } from "components/molecules/UserList";
 import { TablesUserList } from "components/molecules/TablesUserList";
 
+import { BackButton } from "components/atoms/BackButton";
+
 import { TABLES } from "./constants";
 
 import "./ConversationSpace.scss";
 
+export type ConversationSpaceProps = {
+  venue?: WithId<AnyVenue>;
+};
```

Comment:
I think this can be `venue` not `venue?` too.

```suggestion
export interface ConversationSpaceProps {
  venue: WithId<AnyVenue>;
};
```

Reply:
Is there a more specific type we can use here? Either `ConversationSpace` has its own venue type (which looking at `src/types/venues.ts` doesn't seem to be the case), or we can pass `GenericVenue`.

Reply:
@malaikaaiyar You seemingly didn't answer my 2nd question here.

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1761/commits/a87556915dc8a8c2195e0cb6b7c1521187ba0ac7